lib/config/config-initializer.js

```diff
@@ -417,16 +429,62 @@ function askInstallModules(modules, packageJsonExists) {
 function promptUser() {
 
     return inquirer.prompt([
+        {
+            type: "list",
+            name: "purpose",
+            message: "How would you like to use ESLint?",
+            default: "problems",
+            choices: [
+                { name: "To check syntax only", value: "syntax" },
```

Comment:
I think the "check syntax only" option is a fairly niche use case -- most projects would benefit from at least using `eslint:recommended`. Additionally, some users might not understand what "check syntax" means (it means "only report syntax errors" rather than "report bug-prone syntactic patterns"), or they might not have a clear understanding of how syntax errors differ from semantic errors.

In your view, is this preference sufficiently common that it's worth having an option for? I don't feel very strongly about it, but I'd prefer either:

* Removing it from the `--init` options, or
* Moving it to the bottom of the three options here (if we wanted to retain ordering consistency, we could also just reverse the order of these options entirely).

Reply:
We got some direct feedback a while ago that it wasn't clear ESLint could be used just for syntax checking. While it may be a niche case, I don't think it hurts anything to include it here. This way, new users can easily see that ESLint can be used for a number of different purposes.

I don't see the purpose of reversing the order of the options. Right now, there is a logical progression of options that include more functionality, and either way the common case is in the middle and the default.

As it is, the default selection in this menu is to check syntax and find problems